musicdao/build.gradle

```diff
@@ -43,19 +61,27 @@ android {
     testOptions {
         unitTests.returnDefaultValues = true
     }
+
     packagingOptions {
         resources {
             excludes += ['META-INF/DEPENDENCIES', 'META-INF/LICENSE', 'META-INF/LICENSE.txt', 'META-INF/license.txt', 'META-INF/NOTICE', 'META-INF/NOTICE.txt', 'META-INF/notice.txt', 'META-INF/ASL2.0', 'META-INF/*.kotlin_module']
         }
     }
 
-    namespace 'com.example.musicdao'
+    composeOptions {
+        kotlinCompilerVersion kotlin_version
+        kotlinCompilerExtensionVersion "1.3.1"
```

Comment:
Isn't this an old version?

Reply:
Latest compatible version with Kotlin 1.7.10 (which we are using) is 1.3.1.
https://developer.android.com/jetpack/androidx/releases/compose-compiler

```
e: java.lang.Error: Something went wrong while checking for version compatibility between the Compose Compiler and the Kotlin Compiler.  It is possible that the versions are incompatible.  Please verify your kotlin version  and consult the Compose-Kotlin compatibility map located at https://developer.android.com/jetpack/androidx/releases/compose-kotlin
```